### PR TITLE
Update intro suggesting deprication of Archetypes

### PR DESCRIPTION
**Pull request by flipmcf**
First stab at new intro paragraph.  I'm not in love with it.

Maybe we should remove the suggestions that "Dexterity is an alternative to Archetypes" and update our history to suggest that Plone embraced Dexerity as of version 5 and deprecated it at 5.2

### CODE REVIEW

**Comment by flipmcf** [comment]
Agreement signed.

File: docs/intro.rst [hunk 1]
```diff
@@ -54,6 +54,4 @@ Some of the main differences include:
...
 
-All that said, Archetypes is still an older and more mature framework. Dexterity is a viable alternative to Archetypes in many projects, but if you are happy with Archetypes, you should not immediately rush to rewrite your types to use Dexterity.
-
-There are also some things that Dexterity does not yet support, or, more commonly, services that Plone ships with that currently assume all content objects are built using Archetypes. The current list of "gaps" can be found in the Dexterity issue tracker. You should take a look at this before deciding whether Dexterity will work for you. If in doubt, don't hesitate to write to the Dexterity mailing list and ask for advice.
+As of version 5 of Plone, Dexterity is the preferred way of creating content types.  Additionally, Archetypes was removed from Plone core in 5.2.  It can still be added to Plone to support Archetypes-based add-ons, but it will not function when running Plone using Python 3.   If you are considering upgrading to Plone 5 and Python 3, you are reading the right documentation.
```
Comment (jensens) [inline docs/intro.rst]: Content is fine, but please follow the doc style for all new edits; here: One sentence per line. 
https://docs.plone.org/develop/styleguide/documentation.html

Comment (flipmcf) [reply docs/intro.rst]: thanks!

Comment (flipmcf) [reply docs/intro.rst]: https://docs.plone.org/about/contributing/documentation_styleguide.html is correct link to documentation styleguide.

**Comment by flipmcf** [comment]
Force push reset CHANGES.rst back.

**Pull request merged**